keadm/app/cmd/edge/join.go

```diff
@@ -102,14 +106,47 @@ func addJoinOtherFlags(cmd *cobra.Command, joinOptions *types.JoinOptions) {
 
 	cmd.Flags().StringVarP(&joinOptions.EdgeNodeID, types.EdgeNodeID, "i", joinOptions.EdgeNodeID,
 		"KubeEdge Node unique idenfitcation string, If flag not used then the command will generate a unique id on its own")
+
+	cmd.Flags().Uint16Var(&joinOptions.WebSocketPort, types.WebSocketPort, joinOptions.WebSocketPort,
+		"WebSocket port of kubeEdge edgecontroller, default: 10000")
+
+	cmd.Flags().Uint16Var(&joinOptions.QuicPort, types.QuicPort, joinOptions.QuicPort,
+		"Quic port of kubeEdge edgecontroller, default: 10001")
+
+	cmd.Flags().StringVar(&joinOptions.EdgeYaml.EdgeHub.Controller.Protocal, types.Protocal, joinOptions.EdgeYaml.EdgeHub.Controller.Protocal,
+		"Protocal between edge_core and edgecontroller, support websocket and quic , default: websocket")
+
+	joinOptions.EdgeYaml.EdgeHub.WebSocket.URL = fmt.Sprintf("wss://%s:10000/e632aba927ea4ac2b575ec1603d56f10/%s/events",
```

Comment:
This string is used in multiple places, Hence define a constant string and used it in different places.

Reply:
ok